src/net/dmytro/homework4/Cat.java

```diff
@@ -0,0 +1,32 @@
+package net.dmytro.homework4;
+
+public class Cat extends Animal {
+private String catName;
+private static int count = 0;
+
+    public Cat(String catName) {
+        this.catName = catName;
+    }
+
+    @Override
+    void ran(int maxRan) {
+        if (maxRan >= 0 ) {
+            System.out.println("Cat " + catName +" run " + maxRan + " м");
+        }
+        count++;
```

Comment:
it should be in another place, you count cats only if they run, but should count when create, same with dogs

and in the end you count Animal as dogs + cats, but better to add counter in Animals class